exporter/src/main/as/flump/export/EditFormatsWindow.mxml

```diff
@@ -41,20 +41,6 @@
         <s:GridColumn dataField="scaleFactorsString" headerText="Scale Factors"/>
         <s:GridColumn dataField="textureBorder" headerText="Texture Border"/>
         <s:GridColumn dataField="maxAtlasSize" headerText="Max Atlas Size"/>
-        <s:GridColumn dataField="optimize" headerText="Optimize">
-          <s:itemEditor>
-            <fx:Component>
-              <s:ComboBoxGridItemEditor>
-                <s:dataProvider>
-                  <s:ArrayList>
-                    <fx:String>Memory</fx:String>
-                    <fx:String>Speed</fx:String>
-                  </s:ArrayList>
-                </s:dataProvider>
-              </s:ComboBoxGridItemEditor>
-            </fx:Component>
-          </s:itemEditor>
-        </s:GridColumn>
```

Comment:
I tested ATF textures extensively in my last project. They look fine ("good enough") for 3D projects, I'm sure - but for UI components built up in Flump, they are completely unacceptable. The lossy compression looks terrible when you're looking for nice clean lines all over the place. 

I'm also well aware of the draw call problem, but sometimes having 2 2k and 1 1k texture with more draw calls is way more acceptable than one 4k texture that is 30% empty.

To be clear, I'm not arguing against this PR in general - the packing algorithm in Flump is not great, and in particular can be painfully slow under some circumstances. I'm just worried that losing this setting will have a negative performance impact in some highly tuned casses.